cfme/tests/infrastructure/test_kubevirt.py

```diff
@@ -3,23 +3,94 @@
 import pytest
 
 from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.generators import random_vm_name
+from cfme.utils.log import logger
 from cfme.utils.update import update
 
+
 pytestmark = [
-    pytest.mark.provider([KubeVirtProvider], scope="module")
+    pytest.mark.provider([KubeVirtProvider]),
+    pytest.mark.usefixtures('setup_provider')
 ]
 
 
-def test_k6t_provider_crud(provider):
-    """Tests provisioning via PXE
+@pytest.fixture
+def vm_name():
```

Comment:
This is overkill and doesn't need to be a fixture, its just a method call to generate a name.

Reply:
corrected